lib/getstats.js

```diff
@@ -316,6 +323,38 @@ function chromeGetTrackStats(peerConnection, track) {
   });
 }
 
+/**
+ * Get the standardized statistics for a particular MediaStreamTrack in Edge.
+ * @param {RTCPeerConnection} peerConnection
+ * @param {MediaStreamTrack} track
+ * @returns {Promise.<StandardizedTrackStatsReport>}
+ */
+function edgeGetTrackStats(peerConnection, track) {
+  return waitForEdgeIceCompletion(peerConnection).then(function() {
+    return peerConnection.getStats(track);
+  }).then(function(response) {
+    return standardizeEdgeAndFirefoxStats(response);
+  });
+}
+
+function waitForEdgeIceCompletion(peerConnection) {
+  const iceConnectionStatesToWait = new Set(['checking', 'connected', 'new']);
+  if (!iceConnectionStatesToWait.has(peerConnection.iceConnectionState)) {
```

Comment:
I think the following would be better—that way we don't need to construct a Set.

```js
if (peerConnection.iceConnectionState === 'new'
  || peerConnection.iceConnectionState === 'checking'
  || peerConnection.iceConnectionState === 'connected') {
  // ...
}
```